src/stale-prs.ts

```diff
@@ -340,6 +342,30 @@ export class StalePrFinder {
 
     return ret;
   }
+
+  /**
+   * Returns true if the PR has been reviewed or commented on by a core member.
+   */
+  private async memberReviewed(pull_number: number): Promise<Boolean> {
+    
+    const comments = (await this.client.paginate(this.client.rest.issues.listComments, { ...this.repo, issue_number: pull_number, }));
+    
+    // Comments by team members
+    // Filtering out instances where submitted_at is empty
+    const memberComments = comments.filter(r => r.author_association === 'MEMBER').filter(r => r.submitted_at);
+
+    if (!(memberComments.length > 0)) {
+      const reviews = (await this.client.paginate(this.client.rest.pulls.listReviews, { ...this.repo, pull_number, }));
+
+      // Reviews by team members
+      // Filtering out instances where submitted_at is empty
+      const memberReviews = reviews.filter(r => r.author_association === 'MEMBER').filter(r => r.submitted_at);
```

Comment:
I think I can remember that `author_association` is not visible if the calling account is not part of the target organization; and our bots are not members of the target organization (mandated by our security department).

Are we using this condition elsewhere already?

Reply:
Yes, it's being used in the `reviewState()` method which is where I copied this line from.